src/DotnetRelease/PatchReleaseOverviewRecords.cs

```diff
@@ -100,9 +104,15 @@ public record SdkComponent(
     [property: Description("The version of Visual Studio that includes this component version.")]
     string VSVersion,
 
+    [property: Description("The version of Visual Studio on MacOS that includes this component version.")]
```

Comment:
I'm not in favor of adding these. At this point, these properties are pure legacy. What is the motivation for keeping them?

Reply:
I did not intentionally add those. I only rebased what I already had onto your latest state and it seems they were re-added by that. I will remove them once I‘m back in office mid next week. Thanks for noticing!

Reply:
Got it. No worries.